### PR TITLE
Adding e2e testing

### PR DESCRIPTION
**Pull request by vcernomschi**
Adding e2e testing

### CODE REVIEW

File: src/DeepNgToDo/Frontend/js/app/angular/controllers/DeepNgToDoStateController.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 'use strict';
 'format es6';
```
Comment (devs-deep) [inline src/DeepNgToDo/Frontend/js/app/angular/controllers/DeepNgToDoStateController.js]: Codacy Issue found: [Parsing error: Illegal import declaration](https://www.codacy.com/app/MitocGroup/deep-microservices-todo-app/file/1735880808/issues/source?bid=2875840&fileBranchId=2992404#l2)

**Comment by vcernomschi** [comment]
Closed due to proccess out of memory issue in deepify:

```
<--- Last few GCs --->
  531810 ms: Scavenge 1331.3 (1457.1) -> 1331.3 (1457.1) MB, 0.3 / 0 ms (+ 0.8 ms in 1 steps since last GC) [allocation failure] [incremental marking delaying mark-sweep].
  532974 ms: Mark-sweep 1331.3 (1457.1) -> 1331.0 (1457.1) MB, 1163.6 / 0 ms (+ 4.7 ms in 6 steps since start of marking, biggest step 1.1 ms) [last resort gc].
  533937 ms: Mark-sweep 1331.0 (1457.1) -> 1330.4 (1457.1) MB, 963.1 / 0 ms [last resort gc].
<--- JS stacktrace --->
==== JS stack trace =========================================
Security context: 0xa0c7cf37399 <JS Object>
    2: _compile [module.js:~379] [pc=0x3ab017c3d4ae] (this=0x28f5e0696f09 <a Module with map 0x317043d10f99>,content=0x3cad0784a151 <Very long string[5278]>,filename=0x35ba686371e1 <String[183]: /home/travis/build/MitocGroup/deep-microservices-todo-app/src/DeepNgToDo/Backend/src/Task/Create/node_modules/deep-framework/node_modules/deep-resource/lib.compiled/Resource/Action.js>)
    3: .js [modul...
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
```

**Comment by vcernomschi** [comment]
Reopened PR and commented out 4 tests to be able to run e2e tests while the issue https://github.com/MitocGroup/deepify/issues/113 will be fixed

File: test/e2e/General/General.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* global browser */
+/* global protractor */
+/* global element */
```
Comment (devs-deep) [inline test/e2e/General/General.js]: Codacy Issue found: ["element" is defined but never used](https://www.codacy.com/app/MitocGroup/deep-microservices-todo-app/file/2096183354/issues/source?bid=2875840&fileBranchId=2992404#l3)

File: test/e2e/General/General.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+/* global by */
```
Comment (devs-deep) [inline test/e2e/General/General.js]: Codacy Issue found: ["by" is defined but never used](https://www.codacy.com/app/MitocGroup/deep-microservices-todo-app/file/2096183354/issues/source?bid=2875840&fileBranchId=2992404#l4)

**Pull request merged**